code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/message/views/Message.kt

```diff
@@ -105,3 +149,7 @@ internal fun Message(
         onDisposed = onDisposed
     )
 }
+
+@ReadOnlyComposable
+@Composable
+fun getDialogWindow(): Window? = (LocalView.current.parent as? DialogWindowProvider)?.window
```

Comment:
Since this is only used inside this file, can you make this `private`.

Reply:
Removed unnecessary composable function here.